snap/info_snap_yaml.go

```diff
@@ -116,6 +127,8 @@ type socketsYaml struct {
 // InfoFromSnapYaml creates a new info based on the given snap.yaml data
 func InfoFromSnapYaml(yamlData []byte) (*Info, error) {
 	var y snapYaml
+	// Customize hints for the typo detector.
+	y.TypoLayouts.Hint = `use singular "layout" instead of plural "layouts"`
```

Comment:
I know I'm late to the party and this is just a backport. But it would be great if this hint could be encocded into the struct tag like: ` 	TypoLayouts typoDetector `yaml:"layouts,omitempty typo-hints:"..."`. This would mean things are all in the same place. The downside of course is that its no longer i18n able. But at least right now the hint is also not i18n so…